src/eventInput/MouseEventInput.ts

```diff
@@ -23,6 +40,19 @@ export class MouseEventInput extends EventInput {
     return 0;
   }
 
+  protected _getButton(event: MouseEvent): string {
+    switch (event.buttons) {
```

Comment:
I think it would be better to reduce the code by using a literal object.
and There is no need to use it as a method of this class.
and 
```ts
function getEventNameBy(event: MouseEvent): string {
  const buttonCodeMap = { 1: "left", 2: "right", 4: "middle"};
  return buttonCodeMap[event.buttons] || buttonCodeMap[1];
}
// getEventNameBy({buttons: 1}) => "left"
// getEventNameBy({buttons: 2}) => "right"
// getEventNameBy({buttons: 4}) => "middle"
// getEventNameBy() => "left"
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons

MouseEvent.button and MouseEvent.buttons have different values. 
IE 9+ or later is supports MouseEvent.buttons which can also recognize when multiple buttons are pressed at the same time.
For example, if the secondary (2) and auxiliary (4) buttons are pressed simultaneously, the value is 6 (i.e., 2 + 4).

Reply:
ok. I understood. your options is right.